src/api.ts

```diff
@@ -1,7 +1,7 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'http://www.omdbapi.com/?apikey=94993e8b';
```

Comment:
```suggestion
const API_URL = 'https://www.omdbapi.com/?apikey=94993e8b';
```
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/55802630/185566467-b4079896-a19d-4706-8cc1-d24195bcc673.png">